pkgs/development/ocaml-modules/lutils/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildDunePackage, fetchurl, ocaml, findlib, num }:
+
+buildDunePackage rec {
+  pname = "lutils";
+  version = "1.51.2";
+
+  useDune2 = true;
+
+  src = fetchurl {
+    url = "http://www-verimag.imag.fr/DIST-TOOLS/SYNCHRONE/pool/lutils.1.51.2.tgz";
+    sha512 = "f94696be379c62e888410ec3d940c888ca4b607cf59c2e364e93a2a694da65ebe6d531107198b795e80eecc3c6865eedb02659c7e7c4e15c9b28d74aa35d09f8";
+  };
+
+  buildInputs = [
+    ocaml
+    findlib
+    num
```

Comment:
The `num` library must be propagated.

Reply:
Okay, I guess this is because an ocaml program will have to depend on `num` to use `lutils` functions

Reply:
Yes. Moreover, `propagatedBuildInputs` need not be repeated as `buildInputs` (you can remove the `buildInputs` attribute).